src/blob_gc_job.cc

```diff
@@ -287,8 +309,8 @@ void BlobGCJob::BatchWriteNewIndices(BlobFileBuilder::OutContexts& contexts,
     blob_index.EncodeTo(&index_entry);
     // Store WriteBatch for rewriting new Key-Index pairs to LSM
     GarbageCollectionWriteCallback callback(cfh, ikey.user_key.ToString(),
-                                            std::move(original_index));
-    callback.value = index_entry;
+                                            original_index);
```

Comment:
Why remove `move`?

Reply:
It's unnecessary. `move` implies the original value is no longer usable, but we want to keep using it in some cases.

Reply:
Like what case? From what I can see, the value is assigned at every loop and not used after this line.

Reply:
L212, even though during runtime the `blob_index` isn't used afterwards, but semantically it shouldn't be moved out, that's why I change the constructor parameter type. After changing the type, it becomes confusing to keep using `move` at this line.